s3/pkg/service/multipart.go

```diff
@@ -384,8 +385,15 @@ func (s *s3Service) CompleteMultipartUpload(ctx context.Context, in *pb.Complete
 		if part.Etag != in.CompleteParts[i].ETag {
 			log.Errorln("part etag in meta store is not the same with client's part, partno:", i)
 			err = ErrInvalidPart
-			return err
```

Comment:
Hi @sunfch Why donnot check size in the UploadPart API, but check it in the CompleteMultipart API?

Reply:
because object part size should be less 5MB except last part for multipart

Reply:
OK, does AWS S3 do it in the same way?

Reply:
Yes,  pls refer to  https://docs.aws.amazon.com/AmazonS3/latest/dev/qfacts.html

Reply:
> @sfzeng , because object part size should be less 5MB except last part for multipart

You mean, should not be less tha 5MB except last part can be less than 5MB. right?

Reply:
@kumarashit ,  yes, the last part is allowed to be less than 5MB